File: src/python/trackers/create_tracker.py
import os

import easypost
from dotenv import load_dotenv

load_dotenv()
easypost.api_key = os.getenv('EASYPOST_PROD_API_KEY')

tracker = easypost.Tracker.create(
    tracking_code='2756302021',
    carrier='DHLExpress'
)

print(tracker)

File: src/examples/php/trackers/create-tracker.php
<?php

require_once '/Users/jhammond/git/easypost/easypost-tools/vendor/autoload.php';

use Dotenv\Dotenv;
use EasyPost\EasyPost;
use EasyPost\Error;
use EasyPost\Shipment;

$dotenv = Dotenv::createImmutable('/Users/jhammond/git/easypost/easypost-tools');
$dotenv->load();
EasyPost::setApiKey($_ENV['EASYPOST_TEST_API_KEY']);

try {
    $tracker = \EasyPost\Tracker::create([
        "tracking_code" => "EZ1000000001",
    ]);

    echo $tracker;
} catch (Error $exception) {
    echo $exception;
}
